lib/src/collection/multimap.dart

```diff
@@ -30,6 +30,10 @@ abstract class Multimap<K, V> {
       {K Function(dynamic)? key,
       V Function(dynamic)? value}) = ListMultimap<K, V>.fromIterable;
 
+  /// Constructs a new multimap with the same values as [other]
```

Comment:
Please add a period at the end of the doc comment.

Ref: https://dart.dev/guides/language/effective-dart/documentation